modules/electromagnetics/doc/content/source/kernels/ADMatWaveEquation.md

```diff
@@ -0,0 +1,32 @@
+# ADMatWaveEquation
```

Comment:
I don't have a problem renaming and replacing, but I think the name `ADVectorMatReaction` isn't entirely correct since this object assumes that the material property can have a real & imaginary component and the vector has a real & imaginary component. 

I made an object called `ADVectorMatReaction` before that just needed one material property and one vector, but `ADMatWaveEquation` made that object obsolete (in terms of EM physics).

Reply:
ADComplexVectorMatReaction ?
and keep it in EM

Reply:
or ADMatWaveReaction ?

Reply:
I prefer ADMatWaveReaction.